appveyor.yml

```diff
@@ -0,0 +1,27 @@
+build: false
+
```

Comment:
for now, let's add 

```
branches:
 only:
  - dev
```

This will avoid the CI to be triggered for both the PR and the branch